CMakeLists.txt

```diff
@@ -532,7 +532,24 @@ CONFIGURE_FILE(
   ${CMAKE_CURRENT_BINARY_DIR}/gnuradio-runtime/include/gnuradio/config.h
 )
 
-#Re-generate the constants file, now that we actually know which components will be enabled.
+########################################################################
+# Handle the generated constants
+########################################################################
+
+# Use TIMESTAMP to be compatible with reproducible builds
+# and put in in the cache so configure_file sees it
+string(TIMESTAMP BUILD_DATE "%a, %d %b %Y %H:%M:%S")
+set(BUILD_DATE ${BUILD_DATE} CACHE INTERNAL "Build date")
+message(STATUS "Loading build date ${BUILD_DATE} into constants...")
+message(STATUS "Loading version ${VERSION} into constants...")
+
+#double escape for windows backslash path separators
+string(REPLACE "\\" "\\\\" prefix "${prefix}")
+string(REPLACE "\\" "\\\\" SYSCONFDIR "${SYSCONFDIR}")
+string(REPLACE "\\" "\\\\" GR_PREFSDIR "${GR_PREFSDIR}")
```

Comment:
Before, these values were only modified with the scope of gnuradio-runtime/lib and therefore not visible here. I don't have (nor want to setup) a windows box to verify that. Not saying it isn't useful, just noting this is probably changing the result.

Reply:
Uh, oh, maybe they have some use in `gnuradio-runtime/lib` I'll check that. Thanks.